internal/services/cosmos/cosmosdb_account_resource.go

```diff
@@ -110,6 +110,53 @@ func resourceCosmosDbAccount() *pluginsdk.Resource {
 				}, true),
 			},
 
+			"analytical_storage_configuration": {
+				Type:     pluginsdk.TypeList,
+				Optional: true,
+				Computed: true,
+				MaxItems: 1,
+				Elem: &pluginsdk.Resource{
+					Schema: map[string]*pluginsdk.Schema{
+						"schema_type": {
+							Type:     pluginsdk.TypeString,
+							Required: true,
+							ValidateFunc: validation.StringInSlice([]string{
+								string(documentdb.AnalyticalStorageSchemaTypeWellDefined),
+								string(documentdb.AnalyticalStorageSchemaTypeFullFidelity),
+							}, false),
+						},
+					},
+				},
+			},
+
+			"capacity": {
+				Type:     pluginsdk.TypeList,
+				Optional: true,
+				MaxItems: 1,
+				Elem: &pluginsdk.Resource{
+					Schema: map[string]*pluginsdk.Schema{
+						"total_throughput_limit": {
```

Comment:
what are the units here?

Reply:
The unit is RU/s. I added the description of unit in md file.